### ISSUE TITLE
Option to add purgedb into dupReport.rc missing

### ISSUE THREAD

**Issue by dcurrey** [HandyGuySoftware/dupReport#59]
When updating beta to current version I get the error
~~~
  File "./dupReport.py", line 128, in <module>
    canContinue = initOptions()
  File "./dupReport.py", line 65, in initOptions
    if oMgr.readRcOptions() is True:  # Need to restart program (.rc file needs editing)
  File "/home/dcurrey/dupReport-2.1.0_Beta/options.py", line 211, in readRcOptions
    self.options['purgedb'] = self.options['purgedb'].lower() in ('true')   # boolean
KeyError: 'purgedb'

~~~

**Comment by HandyGuySoftware** [comment]
Doh!

**Comment by HandyGuySoftware** [comment]
Fixed in 2.1.0 Beta 1
